skins/oasis/js/Tracking.js

```diff
@@ -60,16 +60,9 @@ jQuery(function ($) {
 		};
 	})();
 
-	/** article **/
+	/** page header (title-area) **/
 
 	(function () {
-		var category = 'article';
-
-		// Not special pages
-		if ($body.hasClass('ns-special')) {
-			return;
-		}
```

Comment:
I don't think we need this condition for any pages—if there's no buttons, there's nothing to track. And moreover, share button is visible on special pages (http://preview.community.wikia.com/wiki/Special:Version).